src/index.ts

```diff
@@ -698,6 +745,7 @@ export {
   extractCSS,
   dontParse,
   webpackDevServer,
+  webpackPluginServe,
```

Comment:
just a quick thought:

- is any product still using `webpack-dev-server` ? If not, we can probably remove it for consistency.
- the same for `reactHotLoader` - the viewer options we provide the better

Reply:
Yeah, that's what I was thinking too. Technically we could drop `webpackDevServer` and `reactHotLoader` and make this a major version.

Reply:
sounds good!

Reply:
Ok, done. As a side effect now we have less dependencies to download.